PublicSDKAPITester/RevenueCatAPI.m

```diff
@@ -5,10 +5,12 @@
 //  Created by Joshua Liebowitz on 6/18/21.
 //
 
+#import <Purchases/RCEntitlementInfo.h>
+#import <Purchases/RCPurchaserInfo.h>
+#import <Purchases/RCPurchases.h>
```

Comment:
would doing `@import Purchases;` or `#import <Purchases/Purchases.h>` cover it? 
I think this app should mimic a regular app as closely as possible

Reply:
I have this change also queued up in another branch

Reply:
https://github.com/RevenueCat/purchases-ios/pull/595